presubmit.py

```diff
@@ -338,24 +339,21 @@ def main() -> int:
     os.chdir(_SRC_ROOT)
     changed_files = get_changed_files()
 
-    if args.command == 'format':
-        success = yapf(changed_files, False)
-        return bool_to_returncode(success)
-
-    if args.command == 'lint':
-        success = lint(changed_files)
-        return bool_to_returncode(success)
-
-    if args.command == 'typecheck':
-        success = pytype(changed_files)
-        return bool_to_returncode(success)
-
-    if args.command == 'licensecheck':
-        success = license_check(changed_files)
-        return bool_to_returncode(success)
-
-    if args.command == 'test_changed_integrations':
-        success = test_changed_integrations(changed_files)
+    command_check_mapping = {
+        'format': yapf,
+        'lint': lint,
+        'typecheck': pytype,
+        'test_changed_integrations': test_changed_integrations
+    }
+    for command, check in command_check_mapping.items():
```

Comment:
Why iterate and continue, why not

command_check_mapping.get(args.command) and just run on it.

Reply:
Done.